src/OpenTelemetry.Exporter.Stackdriver/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * Update OpenTelemetry SDK version to `1.7.0`.
   ([#1486](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1486))
+* Add netstandard2.1 support and update Google Cloud libraries.
```

Comment:
Could you please put exact versions here?

Reply:
Should I specify the exact versions of two Google SDK libs?

Reply:
Yes please. It can be helful for endusers.